scripts/src/docker/ethereum.rs

```diff
@@ -234,53 +231,57 @@ fn transfer_fn(address: clarity::Address, amount: Uint256) -> Vec<u8> {
 
 async fn send_transaction(
     client: Web3<Http>,
-    to: Option<clarity::Address>,
+    to: Option<Address>,
     gas_limit: u64,
     amount: U256,
     data: Vec<u8>,
 ) -> anyhow::Result<TransactionReceipt> {
-    let chain_id = get_chain_id(client.clone()).await?;
-    let tx = clarity::Transaction {
-        nonce: client
-            .eth()
-            .transaction_count(*DEV_ACCOUNT, None)
-            .compat()
-            .await?
-            .as_u64()
-            .into(),
-        gas_price: 0u32.into(),
-        gas_limit: gas_limit.into(),
-        to: to.unwrap_or_default(),
-        value: <[u8; 32]>::from(amount).into(),
-        data,
-        signature: None,
-    };
-    let signed_raw_tx = tx.sign(&*DEV_ACCOUNT_PRIVATE_KEY, Some(chain_id.into()));
-    let serialized_tx = signed_raw_tx
-        .to_bytes()
-        .map_err(|e| anyhow::anyhow!("failed to serialize transaction {:?}", e))?;
-
-    let tx_id = client
-        .send_raw_transaction_with_confirmation(Bytes(serialized_tx), Duration::from_millis(100), 1)
-        .compat()
+    let dev_account = client.eth().coinbase().await?;
```

Comment:
Did you check whether this is the same coinbase all the time? Might as well hardcode it then :)

Reply:
Unfortunately it is generated - and hence different every time. I could not find a way to fix it, besides supplying an account file up-front. But that is cumbersome, so I decided to leave it as is for the moment.